pkg/pkgupdates/pkgupdates.jsx

```diff
@@ -0,0 +1,532 @@
+var $ = require('jquery');
```

Comment:
Yes it should; added (not pushed yet, will do once I addressed all your feedback).